fsharp-backend/src/Wasm/Wasm.fs

```diff
@@ -320,22 +320,39 @@ type EvalWorker =
           )
           Error($"exception: {e.Message}, metdata: {metadata}")
 
-      match args with
-      | Error e -> return Error e
-      | Ok args ->
+      let! result =
+        task {
+          match args with
+          | Error e -> return Error e
+          | Ok args ->
+            try
+              let! result = Eval.performAnalysis args
+              return Ok result
+            with
+            | e ->
+              let metadata = Exception.nestedMetadata e
+              System.Console.WriteLine("Error running analysis in Blazor")
+              System.Console.WriteLine($"called with message: {message}")
+              System.Console.WriteLine(
+                $"caught exception: \"{e.Message}\" \"{metadata}\""
+              )
+              return Error($"exception: {e.Message}, metadata: {metadata}")
+        }
+
+      let serialized =
+        let serialize = Json.Vanilla.serialize
```

Comment:
Is this separated out for any reason?

Reply:
If you're referring to the whole `serialized` binding, I agree that there's some redundancy / noisy code here.

Reply:
Specifically:

```
let serialize = Json.Vanilla.Serialize
try
  serialize result
```

instead of

```
try
  Json.Vanilla.Serialize result
```

Reply:
Yeah it's " quick attempt to ensure the Error serialization didn't somehow differ from the Ok serialization"
Dumb protection from someone updating the serialization of `result` but not updating the serialization of the `Error` a few lines later.

I can simplify if you'd like.